### DISCUSSION TITLE
how to make a delay on dispatch modal

### DISCUSSION THREAD

**Discussion by devgene** [livewire/livewire#8838] [1. Help]
I need to make a redirect after show the modal without any click

       ```
     $this->dispatch('open-model');
            redirect()->to('/profile');
```

please help me to make delay for modal,

i need to make first run modal and then after 10 second redirect to another page

**Comment by PhiloNL** [comment]
You could do something like this instead of `redirect`:

```php
$this->js('setTimeout(() => window.location = "/profile"), 10000)');
```

**Reply by devgene** [reply]
Not redirecting
